SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -48,11 +48,11 @@ void FLocalDeploymentManager::PreInit(bool bChinaEnabled)
 	StartUpWorkerConfigDirectoryWatcher();
 }
 
-void FLocalDeploymentManager::Init(FString RuntimeIPToExpose)
+void FLocalDeploymentManager::Init()
 {
 	// Kill any existing runtime processes.
 	// We cannot attach to old runtime processes as they may be 'zombie' and not killable (even if they are not blocking ports).
-	KillExistingRuntime();
+	SpatialCommandUtils::TryKillProcessWithName(SpatialGDKServicesConstants::RuntimeExe);
```

Comment:
I am always slightly worried some other program will be named runtime.exe, and we will try to kill it :(

Reply:
Yup. I wanted to make it something more specific but turns out it has to be runtime.exe